packages/SwingSet/src/vats/timer/vat-timer.js

```diff
@@ -14,13 +14,19 @@ import {
   prepareSingleton,
 } from '@agoric/vat-data';
 import { makeScalarWeakMapStore } from '@agoric/store';
-import { TimeMath } from './timeMath.js';
+import { TimeMath } from '@agoric/time';
 
 // This consumes O(N) RAM only for outstanding promises, via wakeAt(),
 // delay(), and Notifiers/Iterators (for each actively-waiting
 // client). Everything else should remain in the DB.
 
 /**
+ * @typedef {import('@agoric/time/src/types').Timestamp} Timestamp
```

Comment:
as discussed, this faux import copies the structure but not the documentation.

I think that's acceptable for the scope of this change.